src/main/views/remove-list-search-results.njk

```diff
@@ -12,39 +13,59 @@
+                <table class="govuk-table" data-module="moj-sortable-table">
+                    <thead class="govuk-table__head">
                     <tr class="govuk-table__row">
-                        <td class="govuk-table__cell">{{ item.listTypeName }}</td>
-                        <td class="govuk-table__cell">{{ court.name }}</td>
-                        <td class="govuk-table__cell" data-sort-value="{{ item.contDate | dateWithShortMonthNameToSortValue }}">{{ item.contDate }}</td>
-                        <td class="govuk-table__cell">{{ item.dateRange }}</td>
-                        <td class="govuk-table__cell">{{ item.language | titleCase }}</td>
-                        <td class="govuk-table__cell">{{ item.sensitivity | titleCase }}</td>
-                        <td class="govuk-table__cell govuk-table__cell--numeric">
-                          <a class='unsubscribe-action' href='remove-list-confirmation?artefact={{ item.artefactId }}'>Remove</a>
-                        </td>
+                        <th scope="col" class="govuk-table__header" aria-sort="none">{{ tableHeaders[0] }}</th>
+                        <th scope="col" class="govuk-table__header" aria-sort="none">{{ tableHeaders[1] }}</th>
+                        <th scope="col" class="govuk-table__header" aria-sort="descending">{{ tableHeaders[2] }}</th>
+                        <th scope="col" class="govuk-table__header" aria-sort="descending">{{ tableHeaders[3] }}</th>
+                        <th scope="col" class="govuk-table__header" aria-sort="descending">{{ tableHeaders[4] }}</th>
+                      <th scope="col" class="govuk-table__header" aria-sort="descending">{{ tableHeaders[5] }}</th>
+                        <th scope="col" class="govuk-table__header govuk-table__header--numeric">{{ tableHeaders[6] }}</th>
                     </tr>
-                {% endfor %}
-                </tbody>
-            </table>
+                    </thead>
+                    <tbody class="govuk-table__body">
+                    {% set listCount = 0 %}
+                    {% for item in removalList %}
+                        {% set listCount = listCount + 1 %}
+                        <tr class="govuk-table__row">
+                            <td class="govuk-table__cell">{{ item.listTypeName }}</td>
+                            <td class="govuk-table__cell">{{ court.name }}</td>
+                            <td class="govuk-table__cell" data-sort-value="{{ item.contDate | dateWithShortMonthNameToSortValue }}">{{ item.contDate }}</td>
+                            <td class="govuk-table__cell">{{ item.dateRange }}</td>
+                            <td class="govuk-table__cell">{{ item.language | titleCase }}</td>
+                            <td class="govuk-table__cell">{{ item.sensitivity | titleCase }}</td>
+                            <td class="govuk-table__cell govuk-table__cell--numeric">
```

Comment:
The checkbox I think should be centre'd if possible under the select header. It's currently left aligned

![image](https://github.com/hmcts/pip-frontend/assets/87066931/99f89e83-8060-46a2-bd1a-ab60faa82240)